src/BaywatchOperation.php

```diff
@@ -0,0 +1,238 @@
+<?php
+
+namespace Drupal\baywatch;
+
+use Drupal\Core\Serialization\Yaml;
+use Drupal\Core\File\FileSystemInterface;
+
+
+class BaywatchOperation
+{
+
+  public function remove_purge_lateruntime()
+  {
```

Comment:
We should have linting set up on this project but we don't so I'm apologising in advance for this request...the opening curly brace should be on the same line as the function declaration. If you're using Storm it comes bundled with Drupal's CS.

Reply:
Fixed @GROwen!